.eslintrc

```diff
@@ -18,7 +18,8 @@
         "CDL": true,
         "CQL": true,
         "CXL": true,
-        "cds": true
+        "cds": true,
+        "sap": true
```

Comment:
what is `global.sap`?

Reply:
That's [used by UI5](https://github.com/SAP-samples/cap-sflight/pull/27/files#diff-6b8ed253629bdaaba8c08e8662ffe5de967c55e89865cae93288e239cb86ce2cR3).